test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -83,5 +85,34 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
         }
+
+        [Theory, TraceTest]
+        [MemberData(nameof(DnxSdks))]
+        public void AssemblyInfo(DnxSdk sdk)
+        {
+            // Arrange
+            var solution = TestUtils.GetSolution<DnuPackTests>(sdk, "AssemblyInfo");
+            var project = solution.GetProject("Test");
+
+            sdk.Dnu.Restore(solution.RootPath).EnsureSuccess();
+
+            // Act
+            var result = sdk.Dnu.Pack(project.ProjectDirectory);
+
+            // Assert
+            Assert.Equal(0, result.ExitCode);
+
+            var assemblyPath = Path.Combine(result.RootPath, "Debug", "dnx451", "Test.dll");
+            var assembly = Assembly.LoadFrom(assemblyPath);
+            var attributes = assembly.GetCustomAttributes(true);
+            Assert.Equal(project.Title, attributes.OfType<AssemblyTitleAttribute>().First().Title);
+            Assert.Equal(project.Description, attributes.OfType<AssemblyDescriptionAttribute>().First().Description);
+            Assert.Equal(project.Copyright, attributes.OfType<AssemblyCopyrightAttribute>().First().Copyright);
+            Assert.Equal(project.AssemblyFileVersion.ToString(), attributes.OfType<AssemblyFileVersionAttribute>().First().Version);
+            Assert.Equal(project.Version.ToString(), attributes.OfType<AssemblyInformationalVersionAttribute>().First().InformationalVersion);
+            Assert.Equal(project.Version.Version, assembly.GetName().Version);
+
+            TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
```

Comment:
This statement will be called only if all the asserts pass (asserts throw). Is that expected?